### PR TITLE
Report diagnostic counters as Delta Counters

### PR DESCRIPTION
**Pull request by joannatk**

### CODE REVIEW

File: Wavefront.SDK.CSharp/Common/Metrics/WavefrontSdkMetricsRegistry.cs [hunk 1]
```diff
@@ -202,6 +208,19 @@ public WavefrontSdkCounter Counter(string name)
             return GetOrAdd(name, new WavefrontSdkCounter());
         }
 
+
```
Comment (hanwavefront) [inline Wavefront.SDK.CSharp/Common/Metrics/WavefrontSdkMetricsRegistry.cs]: nit: extra line break

File: Wavefront.SDK.CSharp/Common/Metrics/WavefrontSdkMetricsRegistry.cs [hunk 2]
```diff
@@ -202,6 +208,19 @@ public WavefrontSdkCounter Counter(string name)
...
+        /// under this name, create and register a new delta counter.
+        /// </summary>
+        /// <param name="name">The metric name.</param>
+        /// <returns>A new or pre-existing counter.</returns>
+        public WavefrontSdkDeltaCounter DeltaCounter(string name)
+        {
+            return GetOrAdd(name, new WavefrontSdkDeltaCounter());
+        }
+
+
```
Comment (hanwavefront) [inline Wavefront.SDK.CSharp/Common/Metrics/WavefrontSdkMetricsRegistry.cs]: nit: extra line break

File: Wavefront.SDK.CSharp/Common/Utils.cs [hunk 3]
```diff
@@ -27,6 +27,12 @@ public static string Sanitize(string s)
         {
             var sb = new StringBuilder();
             sb.Append('"');
+            bool isTildaPrefixed = s[0] == 126;
+            bool isDeltaPrefixed = s[0] == 0x2206 || s[0] == 0x0394;
+            bool isDeltaTildaPrefixed = isDeltaPrefixed && s[1] == 126;
+            //Console.Write("string is:{}", s);
+            //Console.Write("istildaPrefixed:{0}, is Delta Prefixed:{1}, isDeltaTildaPrefixed:{2}", isTildaPrefixed, isDeltaPrefixed, isDeltaTildaPrefixed);
```
Comment (hanwavefront) [inline Wavefront.SDK.CSharp/Common/Utils.cs]: remove commented out debugging lines

**Review by hanwavefront — COMMENTED** [review]
a couple minor comments, otherwise lgtm

**Review by hanwavefront — APPROVED** [review]
LGTM

**Review by sushantdewan123 — APPROVED** [review]

**Pull request merged**